src/App.vue

```diff
@@ -858,13 +858,14 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
       // (currently used for dissolution filings only)
       const filingFees = []
       for (const filingData of this.getFilingData) {
-        await PayServices.fetchFilingFees(filingData.filingTypeCode, filingData.entityType, true)
-          .then(res => filingFees.push(res))
-          .catch(error => {
-            console.log('Failed to fetch filing fees, error =', error) // eslint-disable-line no-console
-            // return a valid fees structure
-            filingFees.push(cloneDeep(EmptyFees))
-          })
+        try {
+          const ff = await PayServices.fetchFilingFees(filingData.filingTypeCode, filingData.entityType, true)
+          filingFees.push(ff)
+        } catch (error) {
+          console.log('Failed to fetch filing fees, error =', error) // eslint-disable-line no-console
+          // return a valid fees structure
+          filingFees.push(cloneDeep(EmptyFees))
+        }
```

Comment:
I needed to change this to a try-catch block (instead of inline then-catch) because I mock the function (fetchFilingFees) in the unit tests and the mock apparently doesn't support "catch".